src/crl/interactivesessions/shells/bashshell.py

```diff
@@ -41,9 +41,27 @@ class CdToWorkdirFailed(InteractiveSessionError):
     pass
 
 
+class BashShellTypeError(TypeError):
+    pass
+
+
 @RegisterShell()
 class BashShell(AutoCompletableShell):
-    """InteractiveSession Shell interface for bash."""
+    """InteractiveSession Shell interface for bash.
+
+    Args:
+        cmd (str): command to start bash shell, default: bash
+        confirmation_msg (str): string to expect for confirmation to start bash shell
+        confirmation_rsp (str): expected response to confirmation
+        tty_echo (bool): terminal echo value to be set when started
+        init_env (bool): path to the file to be sourced in init, default: None
+        workdir (bool): change to this directory in start
+        banner_timeout (float): timeout in seconds between lines in the start
+                                banner, default 0.1
```

Comment:
This is Sphinx Napoleon Google style but unfortunately currently documentation generation does not use Napoleon. Thus, it would be preferable to add empty lines between arguments in order to make documentation readable.